Re #11498 Better document Cabal's --semaphore=SEMAPHORE option (#12022)
haskell/cabal: changelog.d/pr-12022.md

> synopsis: Better document, in-app and online, Cabal's `--semaphore=SEMAPHORE` option, post GHC's `semaphore-compat-2.0.0` support

Reviewer: ```suggestion
synopsis: Better document Cabal's `--semaphore=SEMAPHORE` option in light of `semaphore-compat-2.0.0`
```

packages: Cabal
prs: #12022
significance:

description: {

> - Cabal's `--semaphore=SEMAPHORE` option is better documented online and on `--help`.

Reviewer: ```suggestion
- Cabal's `--semaphore=SEMAPHORE` option is better documented in the user's guide and in `--help`.
```
